# Enclose CXXFLAGS and LFLAGS assignments in quotes to avoid compiler errors.

--- Gridcoin-Research-development/gridcoinresearch.pro
+++ Gridcoin-Research-development/gridcoinresearch.pro
@@ -68,8 +68,8 @@

 !win32 {
     # for extra security against potential buffer overflows: enable GCCs Stack Smashing Protection
-    QMAKE_CXXFLAGS *= -fstack-protector-all --param ssp-buffer-size=1
-    QMAKE_LFLAGS *= -fstack-protector-all --param ssp-buffer-size=1
+    QMAKE_CXXFLAGS *= "-fstack-protector-all --param ssp-buffer-size=1"
+    QMAKE_LFLAGS *= "-fstack-protector-all --param ssp-buffer-size=1"
     # We need to exclude this for Windows cross compile with MinGW 4.2.x, as it will result in a non-working executable!
     # This can be enabled for Windows, when we switch to MinGW >= 4.4.x.
 } else {
